src/importer/Program.cs

```diff
@@ -14,6 +14,10 @@ static void Main(string[] args)
             app.HelpOption("-?|-h|--help");
             app.Execute(args);
 
+            var users = new CsvReader().ReadUsers(folderOption.Value());
```

Comment:
I think importing users might need to be a separate operation to importing the ucas data as it happens on a different schedule. Happy to iterate on this when required.